Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -61,9 +61,9 @@ public function getDescription()
      *
      * Token at given index is prepended by attributes.
      *
-     * @param Tokens $tokens  Tokens collection
-     * @param int    $index   token index
-     * @param array  $attribs array of token attributes
+     * @param Tokens                    $tokens  Tokens collection
+     * @param int                       $index   token index
+     * @param array<string, Token|null> $attribs array of token attributes
```

Comment:
`Token[]` is a list of Tokens. Flat one, without taking care of numbers. Here you don't have a list but a hash, with string as index. It is important.
`string[]` is bad as strings here are only the keys, `Tokens[]` is bad as (besides the keys matter) ignore the null possibility